packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js

```diff
@@ -23,6 +23,8 @@ module.exports = {
       'idx.email.verification.required', // mapped to v1 key
       'security.access_denied', // mapped to v1 key in i18nTransformer
       'api.users.auth.error.POST_PASSWORD_UPDATE_AUTH_FAILURE', // mapped to oie alias in i18Transformer
+      'api.authn.poll.error.push_rejected',// mapped to oktaverify.rejected in i18Transformer
+      'authfactor.webauthn.error.assertion_validation_failure' // mapped to authfactor.webauthn.error in i18Transformer
```

Comment:
Did we look into updating our lint rule to read from the `i18nTransformer` object? Would cut down on the touch points for developers.

Reply:
Good point, not yet but I have  a ticket for it https://oktainc.atlassian.net/browse/OKTA-399792. Will follow up